server/routes/api/repos.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+const { updateRecord } = require('./../../gitUtility/initialSeed');
+const { Repository } = require('./../../db/repositories');
+
+const router = express.Router();
+
+// Middleware used to check if authenticated.
+const isAuthenticated = (req, res, next) => {
+    if (req.isAuthenticated()) {
+        return next();
+    }
+    console.log('isAuthenticated: No');
+    res.redirect('/user/login');
+    return 0;
```

Comment:
This function will never reach `return` statement. either it will be redirected if not authenticated or `next()` will be called